ruby-challenges.rb

```diff
@@ -21,6 +34,18 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+def no_vowels string
+    string.delete "aeiouA"
+end 
+
```

Comment:
great approach. Recommend adding the rest of the capital vowels to make it dynamic.